### ISSUE TITLE
Incompatibility with ProtocolLib on purpur 1.19.1

### ISSUE THREAD

**Issue by ILikePlayingGames** [Aust1n46/VentureChat#30]
Venturechat appears to be incompatible with v1.19.1. This error is shown consistently on server startup. I have also tested with protocolled build #583, the last 1.19 build, and the same error was shown.

Server: git-Purpur-1736 (MC: 1.19.1)
ProtocolLib: https://ci.dmulloy2.net/job/ProtocolLib/584/
VentureChat: v3.4.1_1.19_patch

```
[07:41:18 ERROR]: Error occurred while enabling VentureChat v3.4.1_1.19_patch (Is it up to date?)
com.comphenix.protocol.reflect.FieldAccessException: Field index 0 is out of bounds for length 0
        at com.comphenix.protocol.reflect.FieldAccessException.fromFormat(FieldAccessException.java:49) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.reflect.StructureModifier.write(StructureModifier.java:294) ~[ProtocolLib.jar:?]
        at mineverse.Aust1n46.chat.utilities.Format.createPacketPlayOutChat(Format.java:441) ~[VentureChat-3.4.1_1.19_patch.jar:?]
        at mineverse.Aust1n46.chat.command.chat.Edit.<init>(Edit.java:24) ~[VentureChat-3.4.1_1.19_patch.jar:?]
        at mineverse.Aust1n46.chat.command.VentureCommandExecutor.initialize(VentureCommandExecutor.java:113) ~[VentureChat-3.4.1_1.19_patch.jar:?]
        at mineverse.Aust1n46.chat.MineverseChat.onEnable(MineverseChat.java:123) ~[VentureChat-3.4.1_1.19_patch.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[purpur-api-1.19-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[purpur-api-1.19-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:548) ~[purpur-api-1.19-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugin(CraftServer.java:594) ~[purpur-1.19.1.jar:git-Purpur-1736]
        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugins(CraftServer.java:508) ~[purpur-1.19.1.jar:git-Purpur-1736]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:641) ~[purpur-1.19.1.jar:git-Purpur-1736]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:427) ~[purpur-1.19.1.jar:git-Purpur-1736]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:342) ~[purpur-1.19.1.jar:git-Purpur-1736]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1140) ~[purpur-1.19.1.jar:git-Purpur-1736]
        at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:310) ~[purpur-1.19.1.jar:git-Purpur-1736]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
[07:41:18 INFO]: [VentureChat] Disabling VentureChat v3.4.1_1.19_patch
[07:41:18 INFO]: [VentureChat] - Disabling...
[07:41:18 INFO]: [VentureChat] - Disabled Successfully
```

**Comment by saboooor** [comment]
same error here

**Comment by dlee13** [comment]
This happened because the System Chat Packet format changed between 1.19 and 1.19.1.

https://nms.screamingsandals.org/1.19/net/minecraft/network/protocol/game/ClientboundSystemChatPacket.html
int typeId - type of chat message

https://nms.screamingsandals.org/1.19.1/net/minecraft/network/protocol/game/ClientboundSystemChatPacket.html
boolean overlay - clients display the message in the center of the screen or in the normal chat box

**Comment by dlee13** [comment]
Fixed by #32

**Comment by ILikePlayingGames** [comment]
Thank you!
